kiosk/tests/test_login.py

```diff
@@ -0,0 +1,29 @@
+from django.test import Client, TestCase
+from django.urls import reverse
+from django.utils.timezone import timedelta
+
+from core.models.MemberModels import Member
+
+
+class LoginTest(TestCase):
+    def setUp(self):
+        Member.objects.create_member(
+            email='testemail@test.com',
+            rfid='0000000001',
+            membership_duration=timedelta(days=7),
+            password='password'
+        )
+
+    def test_redirect_if_not_logged_in(self):
+        response = self.client.get(reverse('home'))
+        self.assertRedirects(response, '/kiosk/login/')
+
+    def test_user_login(self):
```

Comment:
In the future it might be good to have one assert per test function. Works in this case since the two test cases are so closely related